Update Metadata + Screenshots (Revised)

Ashnidh opened the pull request:
As per the suggestion by @vipulgupta2048 in pull request #23 , I have updated the metadata and renamed the screenshots.

Ashnidh committed:
Create new directory - screenshots

Ashnidh committed:
Add screenshots via upload

Ashnidh committed:
Update activity.info

vipulgupta2048 reviewed (changes requested):
Thanks @Ashnidh 
- [ ] The branch name should be `master` instead of `update-info`
https://github.com/Ashnidh/maze-activity/blob/`update-info`/screenshots/en/1.png

vipulgupta2048 commented:
Also, please don't make new PR's with every change as it becomes very difficult to review pull requests back and forth.

Ashnidh commented:
> Also, please don't make new PR's with every change as it becomes very difficult to review pull requests back and forth.

Ok Sir. Thank you

Ashnidh committed:
Update activity.info

Ashnidh commented:
Thank you. Done.

quozl commented:
Thanks.  Reviewed.

* [x] your _screenshots_ metadata links to your own repository instead of Sugar Labs; this will break once you delete your repository, and it would enable your control of the screenshots without review,
* [ ] none of your screenshots show the critical feature of the Maze activity; collaboration between multiple instances of Sugar, with multiple players shown,
* [x] the first and second screenshots show a bug with calculation of dimensions of the maze on a display resolution of 1366x768; see how there is a wider black border on the left of the maze than on the right; are you willing to fix this in the code, or shall we create a tracking issue for someone else to look at it?
* [x] our other screenshots are mostly in 4:3 aspect ratio, so your 16:9 screenshots would look out of place.

You can work around these last two points; use a virtual machine and set the display resolution of the virtual window to 1024x768 (one of the VESA sizes), then capture the screenshot.

Ashnidh committed:
Replace screenshots via upload

Ashnidh committed:
Replace 1.png

Ashnidh committed:
Update activity.info

Ashnidh commented:
> Thanks. Reviewed.
> 
> * [ ]  your _screenshots_ metadata links to your own repository instead of Sugar Labs; this will break once you delete your repository, and it would enable your control of the screenshots without review,
> * [ ]  none of your screenshots show the critical feature of the Maze activity; collaboration between multiple instances of Sugar, with multiple players shown,
> * [ ]  the first and second screenshots show a bug with calculation of dimensions of the maze on a display resolution of 1366x768; see how there is a wider black border on the left of the maze than on the right; are you willing to fix this in the code, or shall we create a tracking issue for someone else to look at it?
> * [ ]  our other screenshots are mostly in 4:3 aspect ratio, so your 16:9 screenshots would look out of place.
> 
> You can work around these last two points; use a virtual machine and set the display resolution of the virtual window to 1024x768 (one of the VESA sizes), then capture the screenshot.

Sir, 
I have captured screenshots in 4:3 aspect ratio using a virtual machine and replaced them with the older ones. 
I have updated screenshots metadata links to Sugar Labs repository.
Regards

quozl commented:
Thanks.  Good progress.

* [ ] Multiple players aren't shown; there should be more than one smiley face in the maze.  On the other hand, I'm having trouble getting this to work at the moment, so there may be an underlying problem.  There's nothing in logs.

* [ ] The description menu item in the activity menu is missing an icon; that's the element to the right of "Maze activity" and to the left of the sharing button.  Here's what it looks like for me, on Ubuntu 18.04 with Sugar 0.112;

![screenshot_ubb_2018-11-13_11 29 03](https://user-images.githubusercontent.com/2936481/48382987-f423bb00-e737-11e8-9fdb-a663aa1997f9.png)

What version of Sugar and Linux are you using?  Is there anything in /home/$USER/.sugar/default/logs/ that might explain why the icon is missing?

Ashnidh commented:
> Thanks. Good progress.
> 
> * [ ]  Multiple players aren't shown; there should be more than one smiley face in the maze.  On the other hand, I'm having trouble getting this to work at the moment, so there may be an underlying problem.  There's nothing in logs.
> * [ ]  The description menu item in the activity menu is missing an icon; that's the element to the right of "Maze activity" and to the left of the sharing button.  Here's what it looks like for me, on Ubuntu 18.04 with Sugar 0.112;
> 
> ![screenshot_ubb_2018-11-13_11 29 03](https://user-images.githubusercontent.com/2936481/48382987-f423bb00-e737-11e8-9fdb-a663aa1997f9.png)
> 
> What version of Sugar and Linux are you using? Is there anything in /home/$USER/.sugar/default/logs/ that might explain why the icon is missing?

I am getting the same again as shown in the screenshot. I am using Sugar 0.112 on a virtual machine with OS Linux  version openSUSE (64-bit)
Regards

quozl commented:
Thanks.  Is there anything in /home/$USER/.sugar/default/logs/ that might explain why the icon is missing?  In particular look at shell.log and the maze activity log.

Ashnidh commented:
Thanks. 
Please find enclosed the screenshots of data found in /home/liveuser/.sugar/default/logs/ 
Regards

![1](https://user-images.githubusercontent.com/44440524/48418444-56ea7600-e77b-11e8-9be5-3a6c1524082d.png)
![2](https://user-images.githubusercontent.com/44440524/48418445-56ea7600-e77b-11e8-8fb2-34a5f6270a3c.png)
![3](https://user-images.githubusercontent.com/44440524/48418447-57830c80-e77b-11e8-93dc-33b6d74bf7e2.png)
![4](https://user-images.githubusercontent.com/44440524/48418449-57830c80-e77b-11e8-9611-775ee7f6980c.png)
![5](https://user-images.githubusercontent.com/44440524/48418450-57830c80-e77b-11e8-8a09-fd12bd1c10e1.png)
![6](https://user-images.githubusercontent.com/44440524/48418451-581ba300-e77b-11e8-8928-cee681b37a99.png)
![7](https://user-images.githubusercontent.com/44440524/48418452-581ba300-e77b-11e8-9aa7-d1ee121963a3.png)
![8](https://user-images.githubusercontent.com/44440524/48418453-58b43980-e77b-11e8-9bcd-a404eb4c86a5.png)
![9](https://user-images.githubusercontent.com/44440524/48418455-58b43980-e77b-11e8-913c-07376efb5a1d.png)
![10](https://user-images.githubusercontent.com/44440524/48418456-594cd000-e77b-11e8-89be-3413b0e6523c.png)
![11](https://user-images.githubusercontent.com/44440524/48418458-594cd000-e77b-11e8-8457-f435b2c73def.png)

quozl commented:
Thanks.  None of the logs were related to a missing icon.  I've no idea why you have a missing icon.  I don't have a copy of openSUSE and don't have the resources to get one at the moment.

If you wish to diagnose further;
* there should be a file `/usr/share/icons/sugar/scalable/actions/edit-description.svg` or similar that can be read and rendered,
* the [DescriptionItem](https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/activity/widgets.py#L248) class calls ToolButton with the name of the icon.

If something like this is going wrong in the toolkit, it would be good to fix it.  Perhaps it is related to a newer version of GTK+.

Does the same problem happen for you at larger display dimensions?

If you don't have time to work on this, you might try Sugar Live Build or Fedora SoaS instead for making your screenshot.

rhl-bthr commented:
Stale.

@Ashnidh in case you plan to work on it again, please feel free to open a new PR
